FreeAPS/Sources/Modules/Home/View/HomeRootView.swift

```diff
@@ -411,8 +398,8 @@ extension Home {
                         Image(systemName: "person.3.sequence.fill")
                             .symbolRenderingMode(.palette)
                             .foregroundStyle(
-                                !(fetchedPercent.first?.enabled ?? false) ? .green : .cyan,
-                                !(fetchedPercent.first?.enabled ?? false) ? .cyan : .green,
+                                !(state.currentOverride != nil) ? .green : .cyan,
```

Comment:
You have defined a colorset for override profiles but rely on green / cyan here. Shouldn't we be using the same color for the icon, when a profile is active?

Reply:
I just update the last color of the 3 color of the initial logo